inc/utils/html-markup.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
```

Comment:
Maybe this function should live in `inc/components/namespace.php` rather than creating a Utils namespace specifically for html-markup helpers?